app/src/main/java/org/covidwatch/android/data/BaseDao.kt

```diff
@@ -0,0 +1,64 @@
+package org.covidwatch.android.data
+
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Update
+
+interface BaseDao<T> {
```

Comment:
Does this not exist already in Kotlin? Seems like something that should be native

Reply:
I wish but no :)